### PR TITLE
Added more supporters and bug fixes

### LINKED ISSUE #7: Improvements for Landing & Supporters
- [ ] Use a Direct Slack [link](https://join.slack.com/t/rebuildearth/shared_invite/zt-2ogi45jvm-8_cY~zppxzNxG7fuiTivjw) instead of the [subdomain](slack.ohc.network) that redirects to Slack
- [ ] Community Fly-out Menu seems to hide itself unless you move your cursor through the desired path.
- [ ] Add Technology Supporters cypress & browserstack
- [ ] Bug: The header seems to jump around when switching to a different Page, this should be solvable.

### PR DESCRIPTION
**Pull request by shivankacker**
fixes #7
(partially)

### CODE REVIEW

File: components/header.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default function Header(props: {
             name: "Slack",
             description: "Join our Slack community to connect with other contributors.",
             image: "/dropdownicons/slack.jpg",
-            href: "https://slack.ohc.network/"
+            href: "https://join.slack.com/t/rebuildearth/shared_invite/zt-2ogi45jvm-8_cY~zppxzNxG7fuiTivjw"
```
Comment (bodhish) [inline components/header.tsx:59]: Lets use `https://slack.ohc.network` so that we can control the invite.

**Comment by tomahawk-pilot** [comment]
#### 📝 Changed routes:


| [**/supporters**](https://landing-18uqqtoiu-ohcnetwork.vercel.app/supporters) | [**/**](https://landing-18uqqtoiu-ohcnetwork.vercel.app/) |
|:-:|:-:|
|<a href="https://landing-18uqqtoiu-ohcnetwork.vercel.app/supporters">Screenshot of /supporters</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/hDYCZlpHhPi4U1Bk7JKmy6dhtishcKkOTkJUZRDSZeURQeZJl-gwSAASfzEYD11GzEApD1-oE39jv5seSV_o2Q.png">(view full size)</a> |<a href="https://landing-18uqqtoiu-ohcnetwork.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Yj8Lwfs311XpXWlG4xgPJuZZptGYTued4xE4xMvUMa0Kw53wG57EAkQ4_xTL7HjrHYLTW6lyHSmYza-hHMXt4A.png">(view full size)</a> |







Commit f2c7724ddd554ef98280fda49ea0e2e5d17be053 (https://landing-18uqqtoiu-ohcnetwork.vercel.app).

**Review by bodhish — APPROVED** [review]

**Comment by gigincg** [comment]
@shivankacker Why revert the slack url?

**Comment by shivankacker** [comment]
Screenshot 2024-08-24 at 5 02 38 PM

@gigincg reverted on @bodhish 's suggestion

**Comment by bodhish** [comment]
@shivankacker whats the status here?

**Comment by shivankacker** [comment]
@bodhish ready to merge

**Pull request merged**